From a4ea998d367870d93120856c5d6f2b4937bdfdb3 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Sat, 3 Dec 2022 13:02:08 +0200
Subject: [PATCH 2/3] GLODROID: Don't add -ljpeg argument when building libyuv
 as subproject

We don't have sysroot with libdir when building within AOSP.
Instead all dependencies are linked directly as path/lib.so link argument.

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 src/android/meson.build | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/android/meson.build b/src/android/meson.build
index 68646120..9b83acaa 100644
--- a/src/android/meson.build
+++ b/src/android/meson.build
@@ -29,7 +29,6 @@ if not libyuv_dep.found()
          '-Wno-sign-compare',
          '-Wno-unused-variable',
          '-Wno-unused-parameter')
-    libyuv_vars.append_link_args('-ljpeg')
     libyuv = cmake.subproject('libyuv', options : libyuv_vars)
     libyuv_dep = libyuv.dependency('yuv')
 endif
-- 
2.39.2

